spec/spec_helper_acceptance_local.rb

```diff
@@ -28,6 +28,8 @@ def create_repo
 
   # Configure all nodes in nodeset
   c.before :suite do
+    LitmusHelper.instance.apply_manifest(File.read(File.join(__dir__, 'setup_acceptance_node.pp')))
+
     case os[:family]
```

Comment:
I was suggesting to remove the code below and convert it into `setup_acceptance_node.pp`. I think you can safely drop the RH 5 code since that was dropped.